coresdk/src/coresdk/raspi_motor_driver.cpp

```diff
@@ -0,0 +1,179 @@
+//
+//  raspi_motor_driver.cpp
+//  splashkit
+//
+// Created by Aditya Parmar on 14/05/2025.
+// Copyright © 2024 XQuestCode. All rights reserved.
```

Comment:
If you contribute this, then the copyright is covered by the GPL license. You won't be able to copyright this yourself.